Added a comment about the readme creation for new packs (#466)
demisto/content-docs: docs/contributing/marketplace.md

7. The pull request is created and a reviewer is assigned.

>     The documentation for new integration/script/playbook is automatically generated and contains the basic information of it.

Reviewer: please add here that the files to be reviewed will be listed at the pr comment

>     The documentation for new packs automatically generated and it contains the starter information for the integration/script/playbook.

Reviewer: ```suggestion
    The documentation for new integration/script/playbook is automatically generated and contains the basic information of it.
```

    You will now need to review the documentation (README.md) and modify it according to XSOAR standards.

>     Visit the [Documentation](https://xsoar.pan.dev/docs/documentation/readme_file) page for more information.

Reviewer: ```suggestion
    See the [Documentation article](https://xsoar.pan.dev/docs/documentation/readme_file) for more information.
```

7. The pull request is created and a reviewer is assigned.

    The documentation for new integration/script/playbook is automatically generated and contains the basic information of it.

>     The files to be reviewed will be listed at the PR comment
>     You will now need to review the documentation (README.md) and modify it according to XSOAR standards.

Reviewer: ```suggestion
    You will now need to review the documentation (README.md) and modify it according to XSOAR standards.
    The files to be reviewed will be listed at the pull request comment.
```
